test/test_fx_experimental.py

```diff
@@ -1463,8 +1463,10 @@ def test_normalize_operator_exhaustive(self, device, dtype, op):
         # Sorted and one entry on each line to minimize merge conflicts.
         op_skip = {
```

Comment:
cc @Chillee We should kill this list and use regular DecorateInfo skips if possible